port_ocean/core/models.py

```diff
@@ -1,11 +1,16 @@
 from dataclasses import dataclass, field
-from enum import Enum
+from enum import Enum, StrEnum
 from typing import Any
 
 from pydantic import BaseModel
 from pydantic.fields import Field
 
 
+class CreatePortResourcesOrigin(StrEnum):
+    Local = "Local"
```

Comment:
```suggestion
    Ocean = "Ocean"
```

Reply:
❤️  I think it will be more clear for users who run it ( as local can be confusing ) ( I know I suggested the `Local` previously )